ajax.toolkit.php

```diff
@@ -827,11 +827,16 @@ function InitDataModel($sConfigFileName, $bModelOnly = true)
 
 				if (count($aSQLFixesTables) > 0)
 				{
-					$oButtonUpCodeAndDb = ButtonUIBlockFactory::MakeForPrimaryAction("Compile + Update DB tables and views","e","📀 Update iTop code and Database! ", true,"bt_up_code_and_db");
+					$oButtonUpCodeAndDb = ButtonUIBlockFactory::MakeForPrimaryAction("📀 Update iTop code and Database!",null,null, true,"bt_up_code_and_db");
 					$oButtonUpCodeAndDb->SetOnClickJsCode("doApply(true);");
+					$oButtonUpCodeAndDb->SetTooltip("Compile + Update DB tables and views");
 					$oPage->AddSubBlock($oButtonUpCodeAndDb);
 				}
 
+				$oButtonSchema = ButtonUIBlockFactory::MakeForPrimaryAction("⟳ Refresh",null, null,false, "bt_content_schema");
+				$oButtonSchema->SetOnClickJsCode('CheckDBSchema(true);');
+				$oButtonSchema->SetIconClass("fas fa-redo");
```

Comment:
Tried to use this FontAwesome icon, but it doesn't seem to work. Any ideas?

Reply:
@steffunky @Molkobain ?

Reply:
The FA stylesheet doesn't seem to be included in the page

Reply:
Oh.. Is that something that should be?

Reply:
To use the icons through CSS classes yes.

Looks like the font file is already included which might actually be a mistake actually. It should be the stylesheet not the font file. Font file will be brought by the stylesheet.

You can check how it is done in the backoffice and make a PR for the toolkit if you feel like it.

Reply:
Mhh, not sure using the iTop Font Awesome lib inside the toolkit would be a good idea ? That would introduce a dependency to the precise file location inside each iTop version :/

Reply:
Its just `/css/font-awesome/css/all.min.css` right? Do you mean this url changes across versions?

Reply:
To me it would be ok to test for the existence of the file and include it. The toolkit is supposed to work with several versions of iTop as are some extensions, so it seems fair to me.

Reply:
But only as much as any other linked file from iTop core I guess? 

Examples:
- ../images/indicator.gif
- js/jquery.ba-bbq.min.js
- ../images/itop-logo.png

Reply:
Yes, that's why we can consider it as an extension from my PoV. It relies on the iTop version installed and its resources, so it's ok to check if the file exists.